### PR TITLE
feat(fa): Setup xRoad connection from the application system to the finacnial-aid-backend

### PR DESCRIPTION
**Pull request by davidpxn**
# ... 👆

https://app.asana.com/0/1202167941440767/1202177040936159

## What

- Configure the requests to financial-aid-backend to use the correct headers and tokens.
- Add env configurations for the xRoad path to the financial-aid-backend

## Why

So the apis from island.is can talk to financial-aid-backend (hosted at sambandid) via xRoad.

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by MargretFinnboga — APPROVED** [review]
🏆

File: charts/islandis/values.prod.yaml [hunk 1]
```diff
@@ -267,6 +266,7 @@ api:
     XROAD_DRIVING_LICENSE_PATH: 'r1/IS/GOV/5309672079/Logreglan-Protected/Okuskirteini-v1'
     XROAD_DRIVING_LICENSE_V2_PATH: 'r1/IS/GOV/5309672079/Logreglan-Protected/Okuskirteini-v2'
     XROAD_FINANCES_PATH: 'IS/GOV/5402697509/FJS-Public/financeIsland'
+    XROAD_FINANCIAL_AID_BACKEND_PATH: 'IS/MUN/5502694739/samband-sveitarfelaga/financial-aid-backend'
```
Comment (hrafnkellbrimar) [inline charts/islandis/values.prod.yaml:269]: This path is not active yet in XRoad

File: charts/islandis/values.staging.yaml [hunk 2]
```diff
@@ -275,6 +274,7 @@ api:
     XROAD_DRIVING_LICENSE_PATH: 'r1/IS/GOV/5309672079/Logreglan-Protected/RafraentOkuskirteini-v1'
     XROAD_DRIVING_LICENSE_V2_PATH: 'r1/IS/GOV/5309672079/Logreglan-Protected/RafraentOkuskirteini-v2'
     XROAD_FINANCES_PATH: 'IS-DEV/GOV/10021/FJS-Public/financeIsland'
+    XROAD_FINANCIAL_AID_BACKEND_PATH: 'IS-TEST/MUN/5502694739/samband-sveitarfelaga/financial-aid-backend'
```
Comment (hrafnkellbrimar) [inline charts/islandis/values.staging.yaml:277]: This path is not active yet in XRoad

**Review by hrafnkellbrimar — COMMENTED** [review]
The paths I commented on need to be set up in XRoad before this will work.

File: charts/islandis/values.prod.yaml (hunk 1 above)
Comment (hrafnkellbrimar) [reply charts/islandis/values.prod.yaml:269]: This has been added now

File: charts/islandis/values.staging.yaml (hunk 2 above)
Comment (hrafnkellbrimar) [reply charts/islandis/values.staging.yaml:277]: This has been added now

**Review by hrafnkellbrimar — APPROVED** [review]
@Snojo and I had a look

**Review by Snojo — APPROVED** [review]

**Pull request merged**